server/server.coffee

```diff
@@ -5,12 +5,41 @@
 # Use Bluebird promises instead of native ones
 global.Promise = require 'bluebird'
 _ = require '../lib/utils'
+CONFIG = require 'config'
+breq = require 'bluereq'
+{ host:elasticHost, indexes } = CONFIG.elastic
 
-app = require('express')()
-bodyParser = require 'body-parser'
-app.use bodyParser.json()
-app.use bodyParser.urlencoded({ extended: true })
+ensureElasticIndex = ->
+  breq.get elasticHost
+  .then createIndex
+  .catch waitForElastic
 
-app.post '/', require('./post')
+createIndex = ->
+  unless indexes.inventaire
+    return console.log "no inventaire indexes"
```

Comment:
this should probably rather throw an error crashing the process as nothing the rest of the process does't make sens without a known `indexes.inventaire`